src/services/session.service.ts

```diff
@@ -0,0 +1,38 @@
+import Session from "../session/session";
+import { IPromiseToFileSession } from "session/types";
+import { saveSession } from "./redis.service";
+import * as keys from "../session/keys";
+
+/**
+ * Creates a new ptf session and adds a company to the context.
+ * This will override any ptf session that is already in redis.
+ * @param chSession
+ * @param companyNumber
+ */
+const createPromiseToFileSession = async (
+  chSession: Session): Promise<IPromiseToFileSession> => {
+  const ptfSession: IPromiseToFileSession = {
+    company_number: "",
```

Comment:
When we create the new session, do we / should we add an empty company number value? Why bother? This could just be added when a client requests to add it to the PTF session. Alternatively, when we call this function, we have to supply a company number as a parameter.

Reply:
We may not have a company number at this point and to have the ptf session type we need at least one field.

Reply:
Suggests that the create method should be parameterised then with a company number. An empty session essentially serves no value.

Reply:
Parameterised.